.github/workflows/release.yaml

```diff
@@ -17,7 +22,15 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-node@v1
         with:
-          node-version: 14
+          node-version: '${{ env.NODE }}'
+      - name: Set up npm cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-v${{ env.NODE }}-${{ hashFiles('package.json') }}-${{ hashFiles('package-lock.json') }}}
+          restore-keys: |
+            ${{ runner.OS }}-node-v${{ env.NODE }}-${{ hashFiles('package.json') }}-${{ hashFiles('package-lock.json') }}
+            ${{ runner.OS }}-node-v${{ env.NODE }}-
```

Comment:
What's the second line about? See s partly repeated and CI only picks up on the first one

Reply:
It's a fallback if the first one isn't found.

Reply:
Mmh after first run that should not be an issue, I'd optimise for readability IMO if this is not gonna be generally an effective addition in the long run

Reply:
The hash changes frequently thus the cache will be invalidate pretty often. Also, this is just one line and a standard one if you check the official actions/cache README.md.

IMHO it should stay.